Updates

jakejohns opened the pull request:
- Support psr/log 1, 2, or 3
- Support php8 (phpunit was locked to 7)
- Get rid of deprecation warning for dynamic property creation

froschdesign commented on phpunit.xml.dist:
```diff
@@ -1,12 +1,13 @@
-        </whitelist>
-    </filter>
+<?xml version="1.0"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="phpunit.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```
This conflicts with the allowed versions in the Composer configuration, 7 and 9.

The local schema file can be used instead:
```xml
xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

froschdesign commented on composer.json:
```diff
@@ -15,15 +15,15 @@
     },
     "require-dev": {
-        "phpunit/phpunit": "~7.0",
+        "phpunit/phpunit": "~7.0 || ~9.0",
```
[Version 7 of PHPUnit is dead](https://phpunit.de/supported-versions.html), also [version 7 of PHP](https://www.php.net/supported-versions).

jakejohns commented:
@froschdesign I believe addressed your points here:
 https://github.com/atlasphp/Atlas.Cli/compare/880aac77651b8891bace78902346651d12adf742..b24153ae36c73e899e429f57c10a2fca35794853

anyone have any other thoughts on this?
